app/models/event.rb

```diff
@@ -16,6 +16,10 @@ class Event < ApplicationRecord
     where('date = ?', Date.current - NOTIFICATION_PERIOD)
   end
 
+  def upcoming?
+    Event.upcoming.include?(self)
```

Comment:
I think you are trying to lead me to: If we changed the name of the class in the future we wouldn't need to update this method. I am going to take the advice.